ray-operator/controllers/ray/common/service.go

```diff
@@ -139,6 +144,7 @@ func BuildDashboardService(cluster rayiov1alpha1.RayCluster) (*corev1.Service, e
 			Selector: selectorLabels,
 			Ports:    []corev1.ServicePort{},
 			Type:     cluster.Spec.HeadGroupSpec.ServiceType,
+			LoadBalancerIP: cluster.Spec.HeadGroupSpec.LoadBalancerIP,
```

Comment:
```suggestion
```
I'm not sure if it makes sense to include this here in `BuildDashboardService` and in `BuildServeServiceForRayService`. @kevin85421 can you help confirm?